hack/build-go.sh

```diff
@@ -53,9 +53,12 @@ for arg in $args; do
     if [ "${target}" = "test" ]; then
         (cd $arg; go ${target} -v ./...)
     elif [ "${target}" = "install" ]; then
-        (cd $arg; GOBIN=$PWD go ${target} .)
+        eval "$(go env)"
+        ARCHBIN=$(basename $arg)-$(git describe --always)-$GOHOSTOS-$GOHOSTARCH
+        (cd $arg; GOBIN=$PWD go build -o $ARCHBIN)
         mkdir -p bin
-        ln -sf ../$arg/$(basename $arg) bin/$(basename $arg)
+        ln -sf $ARCHBIN $arg/$(basename $arg)
```

Comment:
I think this will break the docker containers at runtime.

From what I see, our dockerfiles will build by copying over the symbolic link, but the actual binary won't be there. I didn't test this, but it doesn't seem like this will work unless the Dockerfiles are updated to pull in the new binary names as well.

Reply:
IIRC then docker will resolve the symlinks before copying content. Let me double check … Works. It is resolving the symlink.

I decided actually against using the nevra named binary, because this is not nice to maintain.

Reply:
oh nice! i learned something new :D